locale/en_GB/LC_MESSAGES/django.po

```diff
@@ -137,22 +137,42 @@ msgstr "Download options"
 msgid "judgment.downloadoptions.pdf.cta"
 msgstr "Download this judgment as a PDF"
 
+#: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
+msgid "press_summary.downloadoptions.pdf.cta"
+msgstr "Download this press summary as a PDF"
+
 #: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
 msgid "judgment.downloadoptions.pdf.description"
 msgstr ""
 "The original format of the judgment as handed down by the court, for "
 "printing and downloading."
 
+#: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
+msgid "press_summary.downloadoptions.pdf.description"
+msgstr ""
```

Comment:
I did look into whether there was a way I could not duplicate all these translations just for the sake of changing "judgment" to "press_summary" but I couldnt yet get it working.

I might try a bit more to do that as it would be much nicer without all this duplication.

but if anyone has any suggestions, please shout 🙏

Reply:
https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#blocktranslate-template-tag 

we might be able to say 
```
{% blocktranslate with doctype=judgment.type %}
{{doctype}}.downloadoptions.pdf.description
{% endblocktranslate %}
```

(Also, the django documentation seems to suggest that we should just be using English strings, rather than our arbitary codes, with hints given via comments https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#comments-for-translators-in-templates)

Reply:
boo